third_party/CMakeLists.txt

```diff
@@ -221,39 +221,45 @@ set(SHADERC_BUILD_FIXES
   )
 endif()
 
-set(CXX_BACK ${CMAKE_CXX_FLAGS})
-SET(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} "${GTEST_BUILD_FIXES}")
-STRING(REGEX REPLACE ";" " " CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
-add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/googletest EXCLUDE_FROM_ALL)
-set(CMAKE_CXX_FLAGS ${CXX_BACK})
+if (${AMBER_ENABLE_TESTS})
+  set(CXX_BACK ${CMAKE_CXX_FLAGS})
+  SET(CMAKE_CXX_FLAGS ${CMAKE_CXX_FLAGS} "${GTEST_BUILD_FIXES}")
+  STRING(REGEX REPLACE ";" " " CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
```

Comment:
Why don't we use only upper case or lower case characters for CMake commands?
Just change `SET, STRING` to `set, string`?

The same for following lines.

Reply:
This is existing, and it currently works so I don't want to change it in this CL. We can do a cleanup later if desired.

Reply:
Ok.